script/release-publish.sh

```diff
@@ -47,6 +47,8 @@ ZIPF_NO_VER="target/universal/apalache.zip"
 TGZF="target/universal/apalache-${VERSION}.tgz"
 TGZF_NO_VER="target/universal/apalache.tgz"
 
+SHA256F="target/universal/sha256sum.txt"
+(cd target/universal && sha256sum apalache.* > "$SHA256F")
```

Comment:
I think the glob here will only match the unversioned `$TGZF_NO_VER` and `$ZIPF_NO_VER` archives, not the versioned `$TGZF` and `$ZIPF`.

Not sure though what the glob for the other two would look like, as this also contains build directories at this point, right?

Reply:
That's a good point. I had assumed that was intended, but now that I think of it I'm not sure why we *wouldn't* want to include hashes for all the archives. 

I think the following should do the trick:

```suggestion
(cd target/universal && sha256sum {apalache*.tgz,apalache*.zip} > "$SHA256F")
```

Reply:
Actually, we don't even have to be that precise, and could get away here just with globing for `apalache`.